go-controller/pkg/types/const.go

```diff
@@ -100,6 +100,9 @@ const (
 	GatewayLBUDP          = "UDP_lb_gateway_router"
 	GatewayLBSCTP         = "SCTP_lb_gateway_router"
 
+	// Loadbalancer options
+	OptionsHairpin = "options:hairpin_snat_ip=" + V4LBHairpinMasquradeIP + " " + V6LBHairpinMasquradeIP
```

Comment:
wait, is V4LBHairpinMasquradeIP defined in a previous commit? If not, you need to define that in this commit as well.

Reply:
Commit history got a little out of wack I will fix